Covscript/README.md

```diff
@@ -0,0 +1,70 @@
+# Covscript
+
+This repo contains a parser and a generator for Covscript, a minimal language used for formulating Covquestions questionaires.
+
+The language is compiled to a subset of JSON-logic. JSON-logic expressions can be converted back to Covscript.
+
+## Syntax and supported ops
+
+A program in Covscript is always a single expression. The result of the expression is the result of the program.
+
+The following literals are supported:
+
+- Numeric constants (`1`, `-2`, `3.5`, `1e+3`)
+- Boolean constants (`true`, `false`)
+- Quote-Delimeted String constants (`"Hello"`)
+- Vaiables (`question3.value`)
+
+The following binary operations are supported:
+
+- Arithmetic operands (`+`, `-`, `*`, `/`)
+- Logical operands (`and`, `or`)
+- Comparisons (`>`, `<`, `<=`, `>=`, `==`, `!=`)
+- In-Operator for lists (`in`)
+- Unary logical negation (`!`)
+
+Also, there is an inline `If`
+
+The operator precedence follow JavaScript or C++ precedence.
```

Comment:
Should we quickly mention the precedence or at least provide a link to an external documentation? Maybe https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Table with the hint that this only applies regarding our supported operators.

Reply:
Done.